content/departments/marketing/blog.md

```diff
@@ -74,7 +79,26 @@ The `description` field is used as an excerpt for your post on the blog the inde
 - The `socialImage` field is optional. Use the full path to image in order to be read properly on Twitter and Facebook. Ideal image size: 1,200 x 628 pixels. <a href="https://sproutsocial.com/insights/social-media-image-sizes-guide/" rel="nofollow" target="_blank">Latest social size guidelines</a>.
 - The `videoID` field is an optional YouTube video ID and will take priority even if the `socialImage` is present. This will generate an inline video preview card when sharing on social media. This is supported for all types of posts; blog, podcast, and release posts.
 
-## Adding images and other media
+## Using the Blog Post Starter Pack
+
+The Content Platform Team maintains a template Markdown file that exposes all of the content components available for use in the blog. You can directly copy and paste components from this file into your post.
+
+Access the template in the About repository: `/content/blogposts/blog-post-starter-pack.md`
+
+## Adding images
+
+Please use a `Figure` component to add images to your post. Optionally, you can include a `caption` to display a description below the image.
+
+<!-- prettier-ignore -->
+```html
+<Figure 
+  src="https://storage.googleapis.com/sourcegraph-assets/blog/default_hero_social.png"
+  alt="Sourcegraph thumbnail"
+  caption="The Sourcegraph logo"
+/>
+```
+
+Please do not add images using `img` tags or `![text](image.png)` methods.
```

Comment:
This is great! Can you add a section for the other missing components here? (`Alert`, `BlockquoteWithBorder`, `TableWrapper`)

Reply:
Sure thing! I have included descriptions for these three here: 9d453a5